### PR TITLE
feat(eventlogs): store it per repository too

### PR DESCRIPTION
**Pull request by sileht**
Fixes MRGFY-1189

Change-Id: I1b0ed25e8e2a467dba797e03e4eb12acb2b125ba

### CODE REVIEW

File: mergify_engine/eventlogs.py [hunk 1]
```diff
@@ -150,6 +152,13 @@ class EventUpdate(EventBaseNoMetadata):
     event: typing.Literal["action.update"]
 
 
+def _get_repository_key(
+    owner_id: github_types.GitHubAccountIdType,
+    repo_id: github_types.GitHubRepositoryIdType,
+) -> str:
+    return f"eventlogs-repository/{owner_id}/{repo_id}"
```
Comment (jd) [inline mergify_engine/eventlogs.py:159]:
```suggestion
    return f"eventlogs/repository/{owner_id}/{repo_id}"
```

**Review by Greesb — APPROVED** [review]

**Comment by Greesb** [comment]
Skip release note and Skip documentation ?

File: mergify_engine/web/api/eventlogs.py [hunk 2]
```diff
@@ -76,3 +76,22 @@ async def get_pull_request_eventlogs(
     return EventLogsResponse(
         events=[e async for e in eventlogs.get(repository_ctxt, pull)]
     )
+
+
+@router.get(
+    "/repos/{owner}/{repository}/events",  # noqa: FS003
+    summary="Get the event logs of a repository",
+    description="Get the event logs of the requested repository",
```
Comment (jd) [inline mergify_engine/web/api/eventlogs.py:84]:
```suggestion
    summary="Get the events log of a repository",
    description="Get the events log of the requested repository",
```

**Review by jd — CHANGES_REQUESTED** [review]

File: mergify_engine/eventlogs.py (hunk 1 above)
Comment (sileht) [reply mergify_engine/eventlogs.py:159]: This will conflict with `events/{owner_id}/{repo_id}/{pull}`

Comment (jd) [reply mergify_engine/eventlogs.py:159]: too sad not having added `/pull/` there I guess 😢

**Review by jd — APPROVED** [review]

**Pull request merged**